# Commuting versus streaming 

This is the first reasonable figure I can find for US commute distances in miles:

https://www.marketwatch.com/story/here-are-the-typical-commutes-for-every-big-metro-area-2015-03-25

There's the commute distance for 'the typical commutes for every big metro area of America'

In [23]:
import csv
import statistics

In [24]:
with open('avg-us-commute-by-distance.csv') as f:
    rows = csv.DictReader(f, delimiter=';')
    commute_distances = []
    for row in rows:
        commute_distances.append(float(row['Typical commute in miles']))
        
commute_distances

[6.1,
 7.1,
 6.7,
 5.9,
 12.8,
 7.0,
 8.6,
 5.6,
 8.6,
 8.0,
 10.1,
 6.0,
 5.4,
 6.5,
 7.3,
 8.0,
 9.7,
 7.5,
 10.0,
 8.7,
 7.8,
 5.9,
 9.0,
 8.6,
 12.2,
 6.6,
 5.9,
 8.5,
 6.4,
 10.4,
 7.0,
 5.6,
 7.2,
 6.9,
 8.0,
 6.0,
 7.3,
 12.2,
 9.2,
 8.7,
 9.1,
 8.9,
 9.1,
 6.9,
 7.2,
 8.2,
 8.8,
 7.6,
 6.8,
 6.4,
 8.9,
 8.6,
 7.4,
 9.5,
 11.0,
 5.0,
 6.2,
 7.7,
 5.9,
 5.7,
 8.6,
 6.0,
 9.1,
 5.3,
 6.6,
 7.8,
 11.4,
 8.1,
 7.1,
 5.5,
 8.5,
 8.7,
 9.1,
 7.4,
 8.0,
 10.0,
 6.5,
 8.8,
 8.5,
 8.0,
 6.4,
 5.2,
 9.0,
 5.6,
 4.7,
 6.5,
 8.5,
 6.0,
 7.3,
 8.0,
 6.6,
 7.6,
 9.1,
 6.7,
 7.1,
 5.8]

In [28]:
statistics.mean(commute_distances)

7.697916666666667

### So, there we have it - about 7.7 miles


Quick and dirty, but at least it's reproducible easily.